src/Components/Web/src/ElementReferenceExtensions.cs

```diff
@@ -17,7 +17,19 @@ public static class ElementReferenceExtensions
         /// </summary>
         /// <param name="elementReference">A reference to the element to focus.</param>
         /// <returns>The <see cref="ValueTask"/> representing the asynchronous focus operation.</returns>
-        public static ValueTask FocusAsync(this ElementReference elementReference)
+        public static ValueTask FocusAsync(this ElementReference elementReference) => elementReference.FocusAsync();
+
+        /// <summary>
+        /// Gives focus to an element given its <see cref="ElementReference"/>.
+        /// </summary>
+        /// <param name="elementReference">A reference to the element to focus.</param>
+        /// <param name="preventScroll">
+        ///     A Boolean value indicating whether or not the browser should scroll the document to bring the newly-focused element into view.
+        ///     A value of false for preventScroll (the default) means that the browser will scroll the element into view after focusing it.
```

Comment:
Btw, doc comments do not preserve any formatting. If you'd like for this to appear in a separate line, you could consider putting this in a `<para>`.

Reply:
Yes. Fix it